### ISSUE TITLE
Additional config parameters

### ISSUE THREAD

**Issue by chrisc0** [fedemotta/yii2-aws-sdk#2]
How can I set additional configuration parameters such as:
'endpoint' => 'http://localhost:8000'
for DynamoDB?

Thanks!

**Comment by fedemotta** [comment]
Sorry for the delay. This should be fixed with release 2.0.

**Comment by Bruno-Meowtel** [comment]
@chrisc0 Sorry to bump this 7 years old issue, but in case anyone's wondering the same, it has, indeed, been addressed in v2.0.

You can specify additional parameters in the 'extra' variable, this is what I've done to use Wasabi (a competitor of Amazon S3, which supports the S3 Protocol):

```
'wasabisdk' => [
    'class' => 'fedemotta\awssdk\AwsSdk',
    'credentials' => [
        'key' => 'xxx',
        'secret' => 'xxx',
    ],
    'extra' => ['endpoint' => 'https://s3.us-west-1.wasabisys.com', 'use_path_style_endpoint' => true],
    'region' => 'us-west-1',
    'version' => 'latest'
]
```
